extensions/xml/xml.language-configuration.json

```diff
@@ -30,5 +30,6 @@
 			"start": "^\\s*<!--\\s*#region\\b.*-->",
 			"end": "^\\s*<!--\\s*#endregion\\b.*-->"
 		}
-	}
+	},
+	"wordPattern": "[:A-Z_a-z\\uC0-\\uD6\\uD8-\\uF6\\uF8-\\u2FF\\u370-\\u37D\\u37F-\\u1FFF\\u200C-\\u200D\\u2070-\\u218F\\u2C00-\\u2FEF\\u3001-\\uD7FF\\uF900-\\uFDCF\\uFDF0-\\uFFFD\\u10000-\\uEFFFF][-:A-Z_a-z\\uC0-\\uD6\\uD8-\\uF6\\uF8-\\u2FF\\u370-\\u37D\\u37F-\\u1FFF\\u200C-\\u200D\\u2070-\\u218F\\u2C00-\\u2FEF\\u3001-\\uD7FF\\uF900-\\uFDCF\\uFDF0-\\uFFFD\\u10000-\\uEFFFF.0-9\\uB7\\u0300-\\u036F\\u203F-\\u2040]*"
```

Comment:
I need to set the `u` flag for the regex and wrap the hex value for the unicode characters in `{}`